package.json

```diff
@@ -14,7 +14,7 @@
     "@fullcalendar/timegrid": "5.10.1",
     "@microsoft/applicationinsights-react-js": "3.2.1",
     "@microsoft/applicationinsights-web": "2.7.1",
-    "luxon": "2.3.0",
+    "luxon": "^1.28.0",
```

Comment:
I reckon keep v2, and we can ditch @fullcalendar/luxon

Reply:
Bizarrely removing the luxon plugin regressed the timezone stuff. Not worth debugging this imo, I'll just leave it

Reply:
rip, all good